integration/discovery/discovery_test.go

```diff
@@ -114,580 +77,713 @@ var _ = Describe("DiscoveryService", func() {
+			configGroups := config.GetChannelGroup().GetGroups()["Application"]
+			Expect(configGroups.GetGroups()["Org1"].GetValues()["AnchorPeers"].GetValue()).ToNot(BeEmpty())
+			Expect(configGroups.GetGroups()["Org2"].GetValues()["AnchorPeers"].GetValue()).ToNot(BeEmpty())
+
+			By("discovering peers with anchors yields complete information")
+			Eventually(nwo.DiscoverPeers(network, org1Peer0, "User1", "testchannel"), network.EventuallyTimeout).Should(ConsistOf(
+				network.DiscoveredPeer(org1Peer0, "_lifecycle"),
+				network.DiscoveredPeer(org2Peer0, "_lifecycle"),
+			))
+			Eventually(nwo.DiscoverPeers(network, org2Peer0, "User1", "testchannel"), network.EventuallyTimeout).Should(ConsistOf(
+				network.DiscoveredPeer(org1Peer0, "_lifecycle"),
+				network.DiscoveredPeer(org2Peer0, "_lifecycle"),
+			))
+		})
 	})
 
-	It("discovers network configuration, endorsers, and peer membership", func() {
-		By("Updating anchor peers")
-		initialHeight := nwo.GetLedgerHeight(network, org1Peer0, "testchannel")
-		network.UpdateChannelAnchors(orderer, "testchannel")
-		// wait for anchor peer config updates to be committed
-		nwo.WaitUntilEqualLedgerHeight(network, "testchannel", initialHeight+2, org1Peer0)
-		//
-		// bootstrapping a peer from snapshot
-		//
-		By("generating a snapshot at current block number on org1Peer0")
-		blockNum := nwo.GetLedgerHeight(network, org1Peer0, "testchannel") - 1
-		submitSnapshotRequest(network, "testchannel", 0, org1Peer0, "Snapshot request submitted successfully")
-
-		By("verifying snapshot completed on org1Peer0")
-		verifyNoPendingSnapshotRequest(network, org1Peer0, "testchannel")
-		snapshotDir := filepath.Join(network.PeerDir(org1Peer0), "filesystem", "snapshots", "completed", "testchannel", strconv.Itoa(blockNum))
-
-		By("adding peer org3Peer0 to the network")
-		org3Peer0 = &nwo.Peer{
-			Name:         "peer0",
-			Organization: "Org3",
-			Channels: []*nwo.PeerChannel{
-				{Name: "testchannel", Anchor: true},
-			},
-		}
-		network.Peers = append(network.Peers, org3Peer0)
+	Describe("basic etcdraft network with anchor peers", func() {
+		BeforeEach(func() {
+			By("generating a network with anchor peers defined and adding a 3rd org")
+			// add org3 with one peer (to generate cryptogen and configtx files)
+			config.Organizations = append(config.Organizations, &nwo.Organization{
+				Name:          "Org3",
+				MSPID:         "Org3MSP",
+				Domain:        "org3.example.com",
+				EnableNodeOUs: true,
+				Users:         2,
+				CA:            &nwo.CA{Hostname: "ca"},
+			})
+			config.Consortiums[0].Organizations = append(config.Consortiums[0].Organizations, "Org3")
+			config.Profiles[0].Organizations = append(config.Profiles[0].Organizations, "Org3")
+			config.Peers = append(config.Peers,
+				&nwo.Peer{
+					Name:         "peer0",
+					Organization: "Org3",
+					Channels: []*nwo.PeerChannel{
+						{Name: "testchannel", Anchor: true},
```

Comment:
do we really have to put anchor: true here? we have the anchors for org1 and org2 and they should be sufficient no? I know it's like in the previous version, just thinking out loud. 

Maybe if there would be some comment that would require a revision, it might be good to change this to false.